src/runtime/link-accounts-flow.js

```diff
@@ -97,11 +102,17 @@ export class LinkCompleteFlow {
       );
       return promise.then(
         response => {
+          deps
+            .eventManager()
+            .logSwgEvent(AnalyticsEvent.ACTION_LINK_CONTINUE, true);
```

Comment:
Not sure CONTINUE is the right word here. But may be it's consistent with other names? But the flow doesn't really continue here - it just completes. As in "the action is done" by this time. There are just UX considerations at this point, I believe.

Reply:
Ideally I would log this when the user clicks the continue button.  Is there a better location to do that?

I am open to a better naming convention.  The intention is that the user clicked the button to proceed to the next step (of account linking)

Reply:
That's the thing: user actually is not proceeding to the next step of account linking here IIUC. The user is all done - they are just acknowledging this fact. So maybe ACTION_LINK_ACK?

Reply:
I definitely would like to log that the user clicked continue when they click continue.  Where can I log that?

Reply:
I think logging it is good. Just meaning of this event is "acknowledgement" of an action, not "continuation" of an action.

Reply:
If the current meaning of the event is acknowledgement then it is being logged in the wrong place.  We need something to be logged when the user clicks the button to continue and initiate the actual linking flow.  Please clarify where I would log this event in code.

Reply:
Ok I might be misunderstanding this to some extent. Let's look at the actual flows.

Reply:
Note that flow.start here will already log EVENT_GOOGLE_UPDATED and IMPRESSION_GOOGLE_UPDATED so we don't need a second thing logging that.  We just need something that logs that the user clicked the continue with linking button

Reply:
We discussed and we will move this logging to the server in a separate pull